src/subscriptions.js

```diff
@@ -18,6 +17,18 @@ import {
 const subscriptions = {};
 const computations = {};
 
+const stopSubscription = action => {
+  if (subscriptions[action.payload]) {
+    const subscriptionId = subscriptions[action.payload].subscriptionId;
+
+    computations[subscriptionId].stop();
+    subscriptions[action.payload].stop();
+
+    delete computations[subscriptionId];
+    delete subscriptions[action.payload];
```

Comment:
Might be less extreme to do 
```javascript
computations[subscriptionId] = undefined;
subscriptions[action.payload] = undefined;
```

Though I don't really see a problem with what you did here other than my personal rule agains the `delete` keyword haha